gcp_variant_transforms/transforms/infer_headers.py

```diff
@@ -223,6 +248,71 @@ def _infer_info_fields(self, variant, defined_headers):
               str(defined_header.get(_HeaderKeyConstants.NUM)),
               str(info_field_value), str(variant))
           infos[info_field_key] = corrected_info
+
+  def _infer_type_info_fields(self, variant, infos, defined_headers):
+    # type: (vcfio.Variant, Dict[str, Info], vcf_header_io.VcfHeader) -> None
+    """Updates `infos` with inferred annotation type info fields.
+
+    All annotation headers in each annotation field are converted to Info header
+    lines where the new ID corresponds to the given annotation field and header,
+    and the new TYPE corresponds to inferred type of the original header. Since
```

Comment:
nit: Why is `TYPE` all caps? If no reason, use `Type` to be consistent with VCF?

Reply:
Please ignore this comment, I suppose you are referring to `VcfParserHeaderKeyConstants.TYPE` and that is fine.